### PR TITLE
COMMERCE-10777 Account Selector fragment

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-10777

Re-sent from https://github.com/liferay-commerce/liferay-portal/pull/3487.

Requires https://github.com/liferay-commerce/liferay-portal/pull/3499 to be merged first.

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
@andrea-ale-sbarra for review

**Comment by smottal** [comment]
ci:test:stop

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:commerce-functional

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Can you also move the buildLang commit to the bottom of the PR?
Thank you!

File: modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java [hunk 1]
```diff
@@ -46,13 +46,72 @@
...
+	@Override
+	public int doStartTag() throws JspException {
+		try {
+			HttpServletRequest httpServletRequest = getRequest();
+
+			CommerceContext commerceContext =
+				(CommerceContext)httpServletRequest.getAttribute(
+					CommerceWebKeys.COMMERCE_CONTEXT);
+
+			_commerceChannelId = commerceContext.getCommerceChannelId();
```
[the remark this answers is not in the document]
Comment (smottal) [reply modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AccountSelectorTag.java:66]: No, this method is already a fetch inside
```
CommerceChannel commerceChannel = _commerceChannelLocalService.fetchCommerceChannelBySiteGroupId(
  _portal.getScopeGroupId(_httpServletRequest));

if (commerceChannel == null) {
  return 0;
}
...
```

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Pull request closed without merging**